connectors/invidious.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import BaseConnector from 'content/base-connector';
+import Utils from 'content/utils';
+import _ from 'underscore';
+
+new class extends BaseConnector {
+    constructor() {
+        super();
+        this.name = 'Invidious';
+        this.prefix = '/com/invidious';
+        Utils.query('video').then(video => {
+            for (let event of ['timeupdate', 'play', 'pause', 'volumechange']) {
+                video.addEventListener(event, () => this.onStateChanged());
+            }
+        });
+
+        this.titleSelector = 'h1';
+        this.mediaSelector = 'video';
+        this.playButtonSelector = '.vjs-play-control';
+    }
+
+    get uniqueId() {
+        const params = new URLSearchParams(location.search.substr(1));
+        return Promise.resolve(params.get('v')
+                               .replace('_', '_u')
+                               .replace('-', '_d'));
+    }
+
+    get controlsInfo() {
+        const hasPrevButton = document.querySelector(this.prevButtonSelector) !== null;
+        return super.controlsInfo.then(controlsInfo => _(controlsInfo).extend({
+            canStop: false,
+            canGoPrevious: hasPrevButton,
```

Comment:
I think `canGoPrevious` and `canGoNext` are always false unless this website has some kind of playlists.